runner.dockerfile

```diff
@@ -0,0 +1,33 @@
+# Use the base image dataprocessor/hdp-cairo:v0.0.2
+FROM dataprocessor/hdp-cairo:v0.0.2
+
+# Set shell to bash and define working directory
+SHELL ["/bin/bash", "-ci"]
+WORKDIR /hdp-runner
+
+# Install necessary dependencies
+RUN apt-get update && apt-get install -y curl jq \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Fetch the latest release tag from GitHub and install HDP
```

Comment:
i know this will cache even tho the release tag had changed to smth else. Should it better to just use hard coded version tag?